benchmark/runner/runner.sh

```diff
@@ -0,0 +1,12 @@
+cd ../exhaustive-processor
+./gradlew :processor:publishToMavenLocal
+cd ..
+mkdir tmp
+cd tmp
+git clone https://github.com/inorichi/tachiyomi.git
+cd tachiyomi
+git checkout 938339690eecdfe309d83264b6a89aff3c767687
+git apply ../../runner/tachi.patch
```

Comment:
If commands below are commented out because you're going to run through command line or a customized launcher, this patch isn't needed. In that case the exhaustive processor doesn't need to be published / installed, either.